### DISCUSSION TITLE
AMS Symbols Support

### DISCUSSION THREAD

**Discussion by cu1ch3n** [typst/typst#3823] [General]
### Description

Hi! I am looking for some [symbols ](https://milde.users.sourceforge.net/LUCR/Math/mathpackages/amssymb-symbols.pdf) in the Latex package amssymb, such as $\Vdash$ ( `||-` ). 
![\Vdash](https://github.com/typst/typst/assets/80438676/054aeef7-e30f-4620-ba14-b6b888a9d8aa)

May I know if there is already some solutions or workarounds? I tried to use #h() to align two symbols to create a new one, but I failed to make a pretty one. Thanks!

### Use Case

It would be useful for those who used to use Latex and AMS symbols.

**Comment by Enivex** [comment]
Please refrain from using issues to ask questions.

You're looking for `sym.forces` (or just `forces` in math mode), `U+22A9`.

**Reply by cu1ch3n** [reply]
ah sorry. I didn't find the symbol. Thanks for answering the question!
